Formula/swift-outdated.rb

```diff
@@ -0,0 +1,20 @@
+class SwiftOutdated < Formula
+  desc "Check for outdated Swift package manager dependencies"
+  homepage "https://github.com/kiliankoe/swift-outdated"
+  url "https://github.com/kiliankoe/swift-outdated.git", tag: "0.5.0", revision: "0b6c3c87dc35796ad0c54f8c7a1d933cd08c540b"
+  license "MIT"
+  head "https://github.com/kiliankoe/swift-outdated.git", branch: "main"
+
+  depends_on xcode: ["13", :build]
+  depends_on macos: :monterey
+  uses_from_macos "swift"
+
+  def install
+    system "make", "build"
+    bin.install ".build/release/swift-outdated"
+  end
+
+  test do
+    system "#{bin}/swift-outdated", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests.

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula